types/uuencode/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for uuencode 0.0
+// Project: https://github.com/zacbarton/node-uuencode#readme
+// Definitions by: Seth Niimi <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/uuencode/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me

Reply:
Updated to use synaesthoery in both places.